clamav-service/src/app.ts

```diff
@@ -112,28 +112,29 @@ const validateApiKey =
       });
     }
   };
-apiRouter.post('',validateApiKey(config.get('server:apiKey')),
+apiRouter.post('/scan',validateApiKey(config.get('server:apiKey')),
   formData.parse({
     uploadDir: os.tmpdir(),
     autoClean: true,
   }),
   formData.union(),
   utils.asyncHandler(async (req: Request, res: Response) => {
+    logger.info('Scanning file for virus');
     const { path, name } = req.body?.file;
     if(!path){
       logger.error('File path is missing in the request body');
       res.status(400).send({
         message: 'File path is missing in the request body',
       });
     }
-    const filePath = fs.realpathSync(PATH.resolve(os.tmpdir(), path));
-    if (!filePath.startsWith(os.tmpdir())) {
+
+    if (!path.startsWith(os.tmpdir())) {
```

Comment:
Any reasons why you removed the real path sync code which fixed an important codeql warning ?

Reply:
`fs.realpathSync(PATH.resolve(os.tmpdir(), path))` results in the wrong location for the file

Reply:
can you put log statements - and do a dry run - I'd like to see the log statements

Reply:
Actually nevermind, I dont know how I got that from.